sdk/keyvault/azure-keyvault-certificates/tests/test_certificates_models.py

```diff
@@ -0,0 +1,16 @@
+from azure.keyvault.certificates import CertificatePolicy
+
+def test_policy_expected_errors():
+    try:
+        cert_policy = CertificatePolicy("issuer-name", "subject-name")
+        assert False, "Error should've thrown since we only have one positional parameter"
```

Comment:
Sure, this would raise if we add a positional parameter to the method. However, so would every call that worked before the change:
```py
>>> def foo(a):
...  print('it worked')
>>> foo(42)
it worked
>>> def foo(a, b):
...  print('it worked')
>>> foo(42)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: foo() missing 1 required positional argument: 'b'
```
Which is to say there's no need to test this explicitly.